dispatcher/templates/statutory_memberslist_edited.jinja2

```diff
@@ -0,0 +1,71 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+{% import "snippets/macros.jinja2" as macros %}
+{{ macros.head("MyAEGEE: A memberslist has been edited for VAR"|replace("VAR", event_name)) }}
+
+<body style="width:100% !important;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%;margin-top:0;margin-bottom:0;margin-right:0;margin-left:0;padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;">
+  <table cellpadding="0" cellspacing="0" border="0" id="backgroundTable" style="margin:0; padding:0; width:100% !important; line-height: 100% !important; border-collapse:collapse; mso-table-lspace:0pt; mso-table-rspace:0pt;background-color: #F9FAFB;"
+    width="100%">
+    <tbody>
+      <tr>
+        <td width="10" valign="top">&nbsp;</td>
+        <td valign="top" align="center">
+          <table cellpadding="0" cellspacing="0" border="0" align="center" style="width: 100%; max-width: 600px; background-color: #FFF; border-collapse:collapse;mso-table-lspace:0pt;mso-table-rspace:0pt;"
+            id="contentTable">
+            <tbody>
+              <tr>
+                <td width="600" valign="top" align="center" style="border-collapse:collapse;">
+                  <table align="center" border="0" cellpadding="0" cellspacing="0" style="background: #F9FAFB;" width="100%">
+                    <tbody>
+                      <tr>
+                        <td align="center" valign="top">
+                          <div style="font-family: &quot;lato&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; line-height: 28px;">&nbsp;</div>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                  <table align="center" border="0" cellpadding="0" cellspacing="0" style="border: 1px solid #E0E4E8;"
+                    width="100%">
+                    <tbody>
+                      <tr>
+                        <td align="center" style="padding: 56px 56px 28px 56px;" valign="top">
+                          <a style="color: #3498DB; text-decoration: none;" href="my.aegee.eu"
+                            target="_blank"><img style="width: 56px; height: 56px; border: 0;" alt="Logo"
+                              src="https://my.aegee.eu/images/logo.png">
+                          </a></td>
+                      </tr>
+                      <tr>
+                        <td align="center" style="padding: 0 56px 28px 56px;" valign="top">
+                          <div style="font-family: &quot;lato&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; line-height: 28px;font-size: 16px; color: #333;">
+                            <strong>"{{ body_name }}"</strong> has edited their memberslist for "{{ event_name }}".
+                            Their new membership fee is "{{ membership_fee }}".
```

Comment:
```suggestion
                            <strong>{{ body_name }}</strong> has edited their memberslist for {{ event_name }}.
                            Their new membership fee is €{{ membership_fee }}.
```

Reply:
not everyone uses euros

Reply:
i don't think we normalise in euros considering the exchange rates are another potential headache for a-eu

Reply:
This is for the fee towards AEGEE-Europe which is normalized in euros